homeassistant/components/apcupsd/binary_sensor.py

```diff
@@ -62,11 +62,19 @@ def __init__(
             )
         self.entity_description = description
         self._data_service = data_service
+        self._attr_available = False
```

Comment:
This makes the entity unavailable during startup, which seems incorrect? It should be available, but unknown?

Reply:
Sorry, I don't have experience with developing HA integrations.
I mostly just adapted [what `xiaomi_miio` was doing](https://github.com/home-assistant/core/blob/d92b423127df0671abb653a48981e550675f6626/homeassistant/components/xiaomi_miio/sensor.py#L991-L1016).
If I understand the source code of `SensorEntity` correctly, `_attr_native_value` is `None` when the entity is first initialized. Is this enough to cause the state to be `unknown` at startup?
Everything seems to work when I remove the line you highlighted, but I can't find a good way to test what the state is during the startup process.

Reply:
> If I understand the source code of SensorEntity correctly, _attr_native_value is None when the entity is first initialized. 

That is correct, which is resulting in an "unknown" state.

> Is this enough to cause the state to be unknown at startup?

It is expected, actually.

> Everything seems to work when I remove the line you highlighted, but I can't find a good way to test what the state is during the startup process.

It is the default, which is already handled correctly.

Reply:
Ok, thanks. So my latest commit should resolve this.